episodes/audience-and-objectives.md

```diff
@@ -0,0 +1,76 @@
+---
+title: 'Target audience and teaching objectives'
+teaching: 30
+exercises: 30
+---
+
+:::::::::::::::::::::::::::::::::::::: questions 
+
+- Why is it essential to understand the target audience before starting the lesson development process?
+- How prior knowledge affects learning?
+- How do learner profiles help define the learning outcomes?
+- What are SMART objectives?
+- How is Bloom’s Taxonomy used in formulating Learning Objectives?
+- How do good objectives help with focus and provide scope for the lesson?
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+::::::::::::::::::::::::::::::::::::: objectives
+
+By the end of this session, participants should be able to...
+
+- Explain the importance of a well-defined target audience.
+- Evaluate learner profiles and make suggestions for improvement.
+- Explain how Bloom’s taxonomy and SMART framework can help define learning objectives.
+- Evaluate lesson objectives and make suggestions for improvements.
+- Explain how the objectives are used to give the lesson more focus.
+
+::::::::::::::::::::::::::::::::::::::::::::::::
+
+:::::::::::::::::::::::::::::::::::::::::::::::::: checklist
+
+## Reading List
+- [CLDT episode: Identifying Your Target Audience](https://carpentries.github.io/lesson-development-training/03-audience.html)
+- [CLDT episode: Defining Lesson Objectives/Outcomes] https://carpentries.github.io/lesson-development-training/05-objectives.html
+- Section 3 (_Formal curriculum design_) of [_Course design: Considerations for trainers - a Professional Guide_](https://doi.org/10.7490/f1000research.1118395.1) by Allegra Via et al, 2020[^1]
+- [Kirschner et al. 2006](https://carpentries.github.io/lesson-development-training/reference.html#litref)
+- [Lujan & DiCarlo 2005](https://carpentries.github.io/lesson-development-training/reference.html#litref)
+- (**Optional**) Chapter 1, How Does Students’ Prior Knowledge Affect Their Learning?, of Ambrose et al. 2010.↩︎
+- (**Optional**) Chapter 2, How Does the Way Students Organize Knowledge Affect Their Learning?, of Ambrose et al. 2010.↩︎
+
+::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
+
+:::::::::::::::::::::::::::::::::::::::::::::::::: discussion
+
+## Discussion Questions
+- What should a good learner profile cover?
+- What are the different ways learner profiles can be used?
+- Can you give examples from your teaching of a situation when prior knowledge impacted learning? Did that make you change the way you teach this particular content?
+- Why is specificity important when describing the target audience?
+- What is your experience using Bloom’s taxonomy to define lesson objectives? Do you find it helpful, and why?
```

Comment:
The Backward Design episode currently includes this discussion question:

> In The Carpentries, we tend to present learning objectives as opposed to outcomes. What distinction, if any, do you see between these two terms? Which do you think is more appropriate? In your opinion, should The Carpentries switch over and start listing "Outcomes" instead of "Objectives" on episodes pages?

Do you think that question is more appropriate for this week, or should we keep it for the BD overview session?

Reply:
I was thinking about it as well. I'd say in the overview, because that's Objectives and Outcomes.

Reply:
:+1: then, if it's ok with you, I would like to adjust the order of those questions in the Backward Design episode. I think the discussion of Objectives vs Outcomes will make more sense after some of the more general discussions of the BD process as a whole.